content/jobs/tanks/dark-knight/basic-guide.md

```diff
@@ -28,7 +31,7 @@ changelog:
 
 # Introduction
 
-Dark Knight (DRK) is one of the four tanks available in Final Fantasy XIV: Endwalker. Its rotation features large bursts of damage, accompanied by spikes in actions per minute (APM) and requires managing both MP and a Blood Gauge, putting many hard-hitting attacks into party raid buff windows. It also has access to one of the most powerful single-target mitigation abilities available in The Blackest Night (TBN).
+Dark Knight (DRK) is one of the four tanks available in Final Fantasy XIV: Dawntrail. Its rotation features large bursts of damage, accompanied by spikes in actions per minute (APM) and requires managing both MP and a Blood Gauge, putting many hard-hitting attacks into party raid buff windows. It also has access to one of the most powerful single-target mitigation abilities available in The Blackest Night (TBN).
```

Comment:
Since there were no tanks released in Endwalker, I figured this was meant to be updated to the latest expansion.